src/javascript_bindings.pyx

```diff
@@ -100,10 +100,14 @@ cdef class JavascriptBindings:
             properties = self.properties
             objects = {}
             for objectName in self.objects:
-                methods = {}
-                for methodName in self.objects[objectName]:
-                    methods[methodName] = None
-                objects[objectName] = methods
+                attrs = {}
+                for name, value in self.objects[objectName].items():
+                    if self.IsValueAllowedRecursively(value) and \
+                            name == '__call__' or not name.startswith('_'):
```

Comment:
A few questions/notes:
1. It seem this changes the previous behavior, because now methods starting with underscore are not allowed?
2. Does the documentation mention that methods and properties starting with underscore are ignored when using SetObject()?
3. What is the reason to disallow methods starting with single underscore? Same reason for properties?
4. I think that the whole check code `if self.IsValueAllowedRecursively(value) and name == '__call__' or not name.startswith('_'):` should be moved to SetObject() body, unless there is a reason to do this at a later time during actual binding/rebinding? Currently JavascriptBindings.objects contains values that may not be binded, but JavascriptBindings.objects is a public property, so someone may be depending on it? In previous version all values in this object were factually binded.
5. Also, not a big deal, but everywhere else strings are enquoted using double quotes.